.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+## code changes will send PR to following users
+@moj124 @Captainmango
```

Comment:
Can you remove me from here? I'm happy to be a contributor. I would like you to verify PRs for learning purposes

Reply:
Yes, will do.